admin-dev/themes/new-theme/js/pages/product/components/filters/FilterDropdown.vue

```diff
@@ -84,16 +84,20 @@
         required: true,
       },
       children: {
-        type: Array,
+        type: Array as PropType<Array<Record<string, any>>>,
         required: true,
       },
       label: {
         type: String,
         required: true,
       },
+      eventEmitter: {
+        type: Object,
+        required: true,
+      },
     },
     mounted() {
-      this.$parent.$on('clearAll', this.clear);
+      this.eventEmitter.on('clearAll', this.clear);
```

Comment:
```suggestion
      this.eventEmitter.on('clearAllCombinationFilters', this.clear);
```
Ok then maybe at least we should use a more specific value to avoid "collision" with other events. And maybe this should be place into the `CombinationEventMap` while youre're at it